crml/nft/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "crml-nft"
+version = "1.0.0"
+authors = ["Centrality Developers <support@centrality.ai>"]
+edition = "2018"
+repository = "https://github.com/cennznet/cennznet"
+description = "CENNZnet NFT module"
+
+[dependencies]
+cennznet-primitives = { path = "../../primitives", default-features = false }
+serde = { version = "1.0.101", optional = true } 
```

Comment:
In cli and some other modules 1.0.102 is used. In my experience, it's safer to have a single version of serder every everywhere. Would you want to uniform them at this stage?

Reply:
will update this